q2_types/feature_data/_format.py

```diff
@@ -100,34 +100,52 @@ class TSVTaxonomyFormat(model.TextFileFormat):
     """
     HEADER = ['Feature ID', 'Taxon']
 
-    def sniff(self):
+    def _check_tsv_tax_format(self, n=None):
```

Comment:
This is a very minor "style" comment: the name of this util method is very similar to the name of the parent format:

```python
TSVTaxonomyFormat._check_tsv_tax_format
```

See the repeated info in there? Maybe instead this could be named something like `_check_n_records`?